test/rack/prx_auth/token_data_test.rb

```diff
@@ -48,6 +44,12 @@
       assert !token.authorized?(789)
     end
 
+    it 'works for namespaced scopes' do
+      assert token.authorized?(123, :ns1, :namespaced)
+      assert !token.authorized?(123, :namespaced)
+      assert token.authorized?(123, :ns1, :admin)
```

Comment:
+100.  This handling of the non-namespaced scopes is pretty slick.

In your description ... you said these were breaking changes?  How so?

Reply:
Update: the `scope` values no longer apply.  One minor fix to make in CMS for that.

Will also need a new method to replace `token.authorized_resources.keys`.  Like `token.authorized_resources_for(scope)`.